### ISSUE TITLE
Peace Candle doesn't stop mob spawns

### ISSUE THREAD

**Issue by Dragnier** [lumien231/Random-Things#270]
MC - 1.12.2
Forge - 14.23.1.2581
RT - 4.1

Picked up a peace candle in a village and put it down in my base. Info in the JEI window about it says it should stop natural mob spawns in a three chunk radius around the candle. That doesn't happen... I still get all the monsters like normal.

**Comment by lumien231** [comment]
Is the Candle Chunk Loaded?

**Comment by Dragnier** [comment]
Yes. I can even be standing there and see mobs spawn within a three chunk radius. I can even leave on a mining trip underground in the same chunk area, teleport back, and have mobs in the radius. 

Question, does the three chunk distance include the center chunk the candle is in making the total area 5x5 or is it not counting the candle chunk making it a 7x7?

Or is it a diameter rather than a radius making it a 3x3?

**Comment by lumien231** [comment]
Doesn't count the one the Peace Candle is in so the Range looks like this: [Imgur](https://imgur.com/a/wm6t6).

Could you post your fml-client-latest log please?

**Comment by Dragnier** [comment]
Is this what you need? https://gist.github.com/Dragnier/6f16f7b6b0239a782bdeaa9fdbb8058d

If it helps, I'm using Chicken Chunks for a chunk loader.

**Comment by lumien231** [comment]
No i need your debug log, it should be called fml-client-latest.log.

**Comment by Chefbarbie** [comment]
[debug.log](https://github.com/lumien231/Random-Things/files/2103417/debug.log)

This is still going on.
Here's the debug.log

**Comment by PokerfaceCactus** [comment]
I know there's not been a response to this for a long time now but we found this happening today as well. 
The peace candles have worked flawlessly for weeks and suddenly stopped working as of today. 
and the only difference is that we have added EnderIO to our modpack.
I suspect that that might be a cause, and i saw EnderIO was also being loaded in the debug log shown above 

(EnderIO also adds a device that has an effect on spawning mobs. it could be that their implementation conflicts with the peace candle) 

Hope this helps at all.

**Comment by drakray** [comment]
Could it be caused by other mods? I seems to have this problem to with modded mobs, specifically Wizards for EBWizardry, and mods from PrimitiveMobs

EDIT: also, it doesn't prevent ZombiePigman spawning from Nether Portal
Playing the 1.12.2 version

**Comment by D-Pterano** [comment]
I have the same issue with the peace candle not stopping mob spawns, but I can say for certain that it is exclusive to mobs from other mods.
PrimitiveMobs in particular is an example but Fire and Ice and a few other mods included in the MC Eternal Modpack.

Minecraft Version: 1.12.2
Forge Version: 14.23.5.2847
MC Eternal Modpack Version: 1.3.7.1

Would it be possible to have the mod check for mobs added by other mods? or perhaps an easier solution would be to include a whitelist/blacklist for the peace candle in the configs, so for instance I could manually add the mob from other mods like "primitivemobs:baby_spider" to the blacklist.
